src/lib/components/shared/text_editor.svelte

```diff
@@ -70,18 +70,57 @@
     };
 
     const handle_keydown = (event: KeyboardEvent) => {
-        if (!show_emoji_picker) return;
-
         if (event.key === "ArrowUp") {
             event.preventDefault();
             active_emoji_index = (active_emoji_index - 1 + emoji_matches.length) % emoji_matches.length;
         } else if (event.key === "ArrowDown") {
             event.preventDefault();
             active_emoji_index = (active_emoji_index + 1) % emoji_matches.length;
         } else if (event.key === "Enter") {
+            // Dont do anything if the emoji picker is not open
+            if (!show_emoji_picker) return;
```

Comment:
we need to add this: `if (!show_emoji_picker) return;` for "ArrowUp" and "ArrowDown" as well?